apps/ensindexer/ponder/src/ensdb-writer-worker.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * This file manages syncing ENSNode metadata:
+ * - ENSIndexer Public Config
+ * - Indexing Status
+ * into the ENSDb.
+ */
+import { secondsToMilliseconds } from "date-fns";
+import pRetry from "p-retry";
+
+import {
+  CrossChainIndexingStatusSnapshot,
+  type Duration,
+  ENSIndexerPublicConfig,
+  IndexingStatusResponseCodes,
+  OmnichainIndexingStatusIds,
+  validateENSIndexerPublicConfigCompatibility,
+} from "@ensnode/ensnode-sdk";
+
+import { EnsDbClient } from "@/lib/ensdb";
+import { ensIndexerClient, waitForEnsIndexerToBecomeHealthy } from "@/lib/ensindexer";
+
+const INDEXING_STATUS_RECORD_UPDATE_INTERVAL: Duration = 1;
+
+/**
+ * ENSDb Writer Worker
+ *
+ * Runs the following tasks:
+ * 1) On application startup, attempt to upsert serialized representation of
+ *    {@link ENSIndexerPublicConfig} into ENSDb.
+ * 2) On application startup, and then on recurring basis,
+ *    following the {@link INDEXING_STATUS_RECORD_UPDATE_INTERVAL}, attempt to
+ *    upsert serialized representation of {@link CrossChainIndexingStatusSnapshot}
+ *    into ENSDb.
+ */
+async function ensDbWriterWorker() {
```

Comment:
```suggestion
async function ensDbMetadataUpsertWorker() {
```

Is that fair? Seems much more accurate.

Assuming so, please rename all the related things / comments / files / etc.